src/widget/wstarrating.h

```diff
@@ -32,7 +32,9 @@ class WStarRating : public QWidget, public WBaseWidget {
     void mouseMoveEvent(QMouseEvent *event);
     void mouseReleaseEvent(QMouseEvent *event);
     void leaveEvent(QEvent *);
-    
+    virtual bool event(QEvent* pEvent);
+    void fillDebugTooltip(QStringList* debug);
```

Comment:
When I add it, the tooltips are not customized anymore.

Here is the example of the pushbutton (without virtual) : https://github.com/mixxxdj/mixxx/blob/master/src/widget/wpushbutton.h#L77-L84